code/sorting/src/insertion_sort/insertion_sort.cs

```diff
@@ -21,17 +21,24 @@ static void Main(string[] args)
             }
             insertsort(arr, 5);
             Console.WriteLine("The Sorted Array is :");
+            // A utility loop to print an array of size 5
             for (i = 0; i < 5; i++)
                 Console.WriteLine(arr[i]); 
             Console.ReadLine();
-        }        
+        }  
+         // Function to sort array 
+    // using insertion sort
         static void insertsort(int[] data, int n)
```

Comment:
Consider renaming this function to `InsertionSort` so that it matches the proper style and is clear to what it does.

Reply:
Made the changes.